src/Schema.php

```diff
@@ -79,6 +82,14 @@ public static function ensureFast404Response() {
     static::removeFromFile($uploads_dir . '/.htaccess', $template);
   }
 
+  /**
+   * Ensures fast 404 responses for missing files in uploads folder.
+   */
+  public static function ensureFrontControllerAccess() {
+    $template = file_get_contents(Plugin::getBasePath() . '/conf/.htaccess.security');
+    Schema::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
```

Comment:
We should stay consistent with the class references. In [`ensureFast404Response()`](https://github.com/netzstrategen/wordpress-core-standards/pull/36/files#diff-8db9667fb5a2702c4e96d6ba0adf79bdR82) we use `static`, which is our default way of referencing methods of the same class, no?